plugins/web-plugin/src/test/java/setting/JettySettings.java

```diff
@@ -30,7 +30,7 @@
     public void startJetty() throws Exception {
     
         if (dunit) {
-            // stop jetty after all tests
+            // stop jetty after all features
             getRuntime().addShutdownHook(new Thread(new Runnable() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)